### ISSUE TITLE
USER STORY: Performance and Accessibility Testing

### ISSUE THREAD

**Issue by Indre-V** [Indre-V/teacup-tales-bookshop#40]
As a **developer**, I want to **ensure that application meets performance and accessibility standards** so that **provides a seamless experience for all users and performs optimally**.

## Acceptance Criteria
- [x] Page load times should be measured and documented using tool Lighthouse.
- [x] Accessibility audits should be conducted using tools Wave or Lighthouse.
- [x] Results and if any issues identified included in README.md.

## Tasks
- [x] Set up accessibility testing tools and environment Wave, to evaluate the accessibility of the application.
- [x] Document accessibility testing results.
- [x] Set up performance testing tools and environment Lighthouse to evaluate page load times and performance metrics.
- [x] Document performance testing results.
